app/src/main/res/values/color_defs.xml

```diff
@@ -71,4 +71,9 @@
   <color name="color_def_dark_mode_question_player_progress_bar_gradient_end">#00C7B6</color>
   <color name="color_def_whitish_grey">#E5E5E5</color>
   <color name="color_def_checkbox_green">#008577</color>
+  <color name="color_def_oppia_dark_grey_border">#666666</color>
+  <color name="color_def_stroke_silver">#777777</color>
+  <color name="color_def_drag_drop_single_item_gradient_start">#FFFFFF</color>
```

Comment:
This color is also defined as `color_def_white `

Reply:
> This color is also defined as `color_def_white `

Done.